inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -1593,6 +1593,36 @@ InferenceEngine::details::CNNLayerCreator::CNNLayerCreator(const std::shared_ptr
         res->params = params;
         return res;
     });
+
+    addSpecificCreator({"VariadicSplit"}, [](const std::shared_ptr<::ngraph::Node>& node,
+                                             const std::map<std::string, std::string>& params) -> CNNLayerPtr {
+        LayerParams attrs = {node->get_friendly_name(), "Split", details::convertPrecision(node->get_output_element_type(0))};
+        auto res = std::make_shared<InferenceEngine::SplitLayer>(attrs);
+        auto castedLayer = std::dynamic_pointer_cast<ngraph::op::VariadicSplit>(node);
+        if (!castedLayer) THROW_IE_EXCEPTION << "Cannot get " << attrs.type << " layer " << attrs.name;
+
+        auto axis_node = castedLayer->input_value(1).get_node_shared_ptr();
+        const auto axis_node_const = ngraph::as_type_ptr<ngraph::op::Constant>(axis_node);
+        if (!axis_node_const) {
+            THROW_IE_EXCEPTION << "Split " << castedLayer->get_friendly_name() << " has no axes as Constant";
+        }
+
+        auto axis = axis_node_const->cast_vector<int64_t>()[0];
+        if (axis < 0) {
+            axis += castedLayer->get_input_shape(0).size();
+        }
+
+        res->params["axis"] = Builder::asString(axis);
+        return res;
+    });
+
+    addSpecificCreator({"ReorgYolo"}, [](const std::shared_ptr<::ngraph::Node>& node,
+                                         const std::map<std::string, std::string>& params) -> CNNLayerPtr {
+        LayerParams attrs = {node->get_friendly_name(), "ReorgYolo", details::convertPrecision(node->get_output_element_type(0))};
```

Comment:
@ilyachur do we need such trivial creators?

Reply:
No, we don't. My mistake : )